airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -331,4 +331,6 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
             affinity=affinity,
             tolerations=tolerations,
             security_context=self._get_security_context(),
+            env_from_configmap_ref=self.kube_config.env_from_configmap_ref,
+            env_from_secret_ref=self.kube_config.env_from_secret_ref
```

Comment:
As suggested in my PR (see: #4772) by @ashb this should go to `secrets` and `configmaps`.

I had basically the same structure as You are showing here, but it was rejected in code review of my PR.